### PR TITLE
Remove truncation functionality from Stepped Tracker

### PR DESCRIPTION
**Pull request by el-dav**
PR to remove truncation functionality from SteppedTracker. Having discussed with accessibility the truncation + tooltip behaviour is not desirable. Instead text should simply wrap

### CODE REVIEW

File: packages/lab/src/stepped-tracker/StepLabel/StepLabel.tsx [hunk 1]
```diff
@@ -1,51 +1,26 @@
...
-
 const withBaseName = makePrefixer("saltStepLabel");
 
-export interface StepLabelProps extends ComponentPropsWithoutRef<"label"> {
-  children: ReactNode;
-}
+export type StepLabelProps = ComponentPropsWithoutRef<"label">;
 
 export const StepLabel = forwardRef<HTMLLabelElement, StepLabelProps>(
   function StepLabel({ children, className, ...rest }, ref?) {
```
Comment (joshwooding) [inline packages/lab/src/stepped-tracker/StepLabel/StepLabel.tsx:14]: Just a question, what is the question mark for?

Comment (el-dav) [reply packages/lab/src/stepped-tracker/StepLabel/StepLabel.tsx:14]: I don't think it does anything here as ForwardedRef can already be null. Not sure why I added it, might have cut and paste the forwardRef from another component

**Review by joshwooding — APPROVED** [review]

**Review by joshwooding — CHANGES_REQUESTED** [review]
This needs a changeset

**Review by origami-z — DISMISSED** [review]

File: .changeset/fresh-readers-sip.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/lab": patch
+---
+
+Removed truncation and tooltip behaviour from SteppedTracker. After re-assessment it was jusdged that simply wrapping advising against long labels and wrapping text where necessary was preferable.
```
Comment (joshwooding) [inline .changeset/fresh-readers-sip.md:5]:
```suggestion
Removed truncation and tooltip behaviour from Stepped Tracker. After re-assessment it was judged that simply wrapping advising against long labels and wrapping text where necessary was preferable.
```

**Review by alycrys — APPROVED** [review]

**Review by lilyvc — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Pull request merged**